Controller/HelperController.php

```diff
@@ -396,10 +399,16 @@ public function retrieveAutocompleteItemsAction(Request $request)
                 $label = $resultMetadata->getTitle();
             }
 
-            $items[] = array(
+            $item = array(
                 'id'    => $admin->id($entity),
                 'label' => $label,
             );
+
+            if (is_callable($resultItemCallback)) {
```

Comment:
You should throw some exception, if user will set to `response_item_callback` something, that is not callback.